tests/suites/test_suite_platform.function

```diff
@@ -84,30 +95,12 @@ void time_delay_milliseconds(int delay_ms)
  */
 void time_delay_seconds(int delay_secs)
 {
-    mbedtls_time_t current = mbedtls_time(NULL);
-    mbedtls_time_t elapsed_secs;
-
-    sleep_ms(delay_secs * 1000);
-
-    elapsed_secs = mbedtls_time(NULL) - current;
-
-    /*
-     * `mbedtls_time()` was defined as c99 function `time()`, returns the number
-     * of seconds since the Epoch. And it is affected by discontinuous changes
-     * from automatic drift adjustment or time setting system call. The POSIX.1
-     * specification for clock_settime says that discontinuous changes in
-     * CLOCK_REALTIME should not affect `nanosleep()`.
-     *
-     * If discontinuous changes occur during `nanosleep()`, we will get
-     * `elapsed_secs < delay_secs` for backward or `elapsed_secs > delay_secs`
-     * for forward.
-     *
-     * The following tolerance windows cannot be guaranteed.
-     * PLEASE DO NOT ENABLE IT IN CI TEST.
-     */
-    TEST_ASSERT(elapsed_secs - delay_secs >= -1 &&
-                elapsed_secs - delay_secs <   4);
-    /* This goto is added to avoid warnings from the generated code. */
-    goto exit;
+    int i;
+    for(i=0;i<3; i++)
```

Comment:
(minor, whitespace) I'd normally expect to see: space after keyword; space either side of binary operator; space after (but not before) `;` in for-loop; opening brace on same line as keyword:

```suggestion
    for (i = 0; i<3; i++) {
```

Reply:
Re-run again